arch/risc-v/src/mpfs/mpfs_usb.c

```diff
@@ -4001,3 +4024,15 @@ void mpfs_usbinitialize(void)
   mpfs_usbuninitialize();
 }
 
+bool mpfs_vbus_detect(void)
+{
```

Comment:
Should maybe go to mpfs_usb.h for the declaration?

Reply:
Yes I can add the declaration there. We pluck it directly via extern but does not hurt to declare it in the header too.